repository.eselect.5.in

```diff
@@ -0,0 +1,169 @@
+.\" -*- coding: utf-8 -*-
+.\" Copyright 2019 Till Schäfer
+.\" Distributed under the terms of the 2-clause BSD license
+.\"
+.TH repository.eselect 5 "31 January 2019" "Gentoo Linux" "eselect"
+.SH NAME
+\fBrepository.eselect\fP - Manage repos.conf via eselect
+.SH SYNOPSIS
+.nf
+.fam C
+\fBeselect\fP \fBrepository\fP [\fBhelp\fP|\fBusage\fP|\fBversion\fP]
+\fBeselect\fP \fBrepository\fP \fBadd\fP \fIname\fP \fIsync-type\fP \fIsync-uri\fP
+\fBeselect\fP \fBrepository\fP \fBdisable\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+\fBeselect\fP \fBrepository\fP \fBenable\fP (\fIname\fP|\fIindex\fP)
+\fBeselect\fP \fBrepository\fP \fBlist\fP [\fB-i\fP]
+\fBeselect\fP \fBrepository\fP \fBremove\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+
+.SH DESCRIPTION
+\fBrepository.eselect\fP is an eselect module for manipulating /etc/portage/repos.conf entries to third party repositories (also known as overlays). This supersedes the need for app-admin/layman to list, configure, and handle synchronizations of alternate repositories except for those which the package manager does not (eg. mercurial, bazaar and g-sorcery in portage).
+.SH ACTION: ADD
+Add a custom repository
+.PP
+\fBeselect\fP \fBrepository\fP \fBadd\fP \fIname\fP \fIsync-type\fP \fIsync-uri\fP
+.PP
+Parameters:
+.TP
+.B
+\fIname\fP
+Name of the repository to add
+.TP
+.B
+\fIsync-type\fP
+Syncer to use (e.g. git, rsync, svn\.\.\.)
+.TP
+.B
+\fIsync-uri\fP
+Sync URI
+.SH ACTION: DISABLE
+Disable specified repositories without removing their contents
+.PP
+\fBeselect\fP \fBrepository\fP \fBdisable\fP [\fB-f\fP] (\fIname\fP|\fIindex\fP)
+.PP
+Parameters:
+.TP
+.B
+\fB-f\fP
+Force potentially dangerous removals. Required for unregistered repositories and those without sync attributes.
+.TP
+.B
+\fIname\fP
+Names of repositories to disable
+.TP
+.B
+\fIindex\fP
+Numeric index of the repository (from 'list' command)
+.SH ACTION: ENABLE
+Enable the specified repositories from the list
+.PP
+\fBeselect\fP \fBrepository\fP \fBenable\fP (\fIname\fP|\fIindex\fP)
+.PP
+Parameters:
+.TP
+.B
+\fIname\fP
+Names of repositories to enable
+.TP
+.B
+\fIindex\fP
+Numeric index of the repository (from 'list' command)
+.SH ACTION: LIST
+List all repositories. The repositories in the listing are suffixed by a symbol, that indicates their
+current state.
+.PP
+\fBeselect\fP \fBrepository\fP \fBlist\fP [\fB-i\fP]
+.PP
+Parameters:
+.TP
+.B
+\fB-i\fP
+Only list installed
+.PP
+State suffixes:
+.TP
+.B
+*
+Repository is enabled
+.TP
+.B
+#
+Repositorys sync information is out of date and needs to be updated (via
+disable/enable) or is customized
+.TP
+.B
+@
+Repository is not listed by name in the official, published list
```

Comment:
s/official, published/remote/. Technically you can use any list you like.

Also, probably something like 'repository with a matching name is ...' would be more correct.